cloud/libvirt/actuators/machine/actuator.go

```diff
@@ -35,6 +37,7 @@ type ActuatorParams struct {
 
 // NewActuator creates a new Actuator
 func NewActuator(params ActuatorParams) (*Actuator, error) {
+	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
You could do this in a `init()` function.

Reply:
Ha, that's even better :) I will update the PR.